src/Support/Debug/Debug.php

```diff
@@ -0,0 +1,985 @@
+            self::$isActive = false;
+
+            return true;
+        }
+
+        echo $exception->getMessage();
+
+        return false;
+    }
+
+    /**
+     * Throws an exception when a notice or warning is raised
+     *
+     * @param int    $severity
+     * @param string $message
+     * @param string $file
+     * @param int    $line
+     * @param array  $context
+     *
+     * @throws ErrorException
+     */
+    public function onUncaughtLowSeverity(
+        int $severity,
+        string $message,
+        string $file,
+        int $line,
+        array $context = []
+    ): void {
+        if (error_reporting() & $severity) {
+            throw new ErrorException($message, 0, $severity, $file, $line);
+        }
+    }
+
+    /**
+     * Sets if files the exception's backtrace must be showed
+     *
+     * @param array $blacklist
+     *
+     * @return $this
+     */
+    public function setBlacklist(array $blacklist): Debug
+    {
+        $area     = $blacklist['request'] ?? [];
+        $subArray = [];
+        $result   = [];
+
+        foreach ($area as $value) {
+            $subArray[mb_strtolower($value)] = 1;
+        }
+
+        $result['request'] = $subArray;
+        $area              = $blacklist['server'] ?? [];
+        $subArray          = [];
+
+        foreach ($area as $value) {
+            $subArray[mb_strtolower($value)] = 1;
+        }
+
+        $result['server'] = $subArray;
+        $this->blacklist  = $result;
+
+        return $this;
+    }
+
+    /**
+     * Sets if files the exception's backtrace must be showed
+     *
+     * @param bool $showBackTrace
+     *
+     * @return $this
+     */
+    public function setShowBackTrace(bool $showBackTrace): Debug
+    {
+        $this->showBackTrace = $showBackTrace;
+
+        return $this;
+    }
+
+    /**
+     * Sets if files must be completely opened and showed in the output
+     * or just the fragment related to the exception
+     *
+     * @param bool $showFileFragment
+     *
+     * @return Debug
+     */
+    public function setShowFileFragment(bool $showFileFragment): Debug
+    {
+        $this->showFileFragment = $showFileFragment;
+
+        return $this;
+    }
+
+    /**
+     * Set if files part of the backtrace must be shown in the output
+     *
+     * @param bool $showFiles
+     *
+     * @return Debug
+     */
+    public function setShowFiles(bool $showFiles): Debug
+    {
+        $this->showFiles = $showFiles;
+
+        return $this;
+    }
+
+    /**
+     * Change the base URI for static resources
+     *
+     * @param string $uri
+     *
+     * @return $this
+     */
+    public function setUri(string $uri): Debug
+    {
+        $this->uri = $uri;
+
+        return $this;
+    }
+
+    /**
+     * Escapes a string with htmlentities
+     *
+     * @param mixed $value
+     *
+     * @return string
+     */
+    protected function escapeString($value): string
+    {
+        if (true === is_string($value)) {
```

Comment:
If my logic is correct, then `$value` is always `string` type. Because this condition check if `string` and if not `return $value`, and signature of method tells that it always return `string`.

Reply:
Corrected it. Thanks. 

This component will need to be refactored for sure for v6